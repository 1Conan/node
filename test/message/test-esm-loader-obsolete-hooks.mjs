// Flags: --no-warnings --throw-deprecation --experimental-loader ./test/fixtures/es-module-loaders/hooks-obsolete.mjs
/* eslint-disable node-core/require-common-first, node-core/required-modules */

await import('whatever');
