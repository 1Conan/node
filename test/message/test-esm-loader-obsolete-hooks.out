node:internal/process/warning:*
        throw warning;
        ^

DeprecationWarning: Obsolete loader hook(s) supplied and will be ignored: dynamicInstantiate, getFormat, getSource, transformSource
    at Function.pluckHooks (node:internal/modules/esm/loader:*:*)
    at ESMLoader.addCustomLoaders (node:internal/modules/esm/loader:*:*)
    at initializeLoader (node:internal/process/esm_loader:*:*)
    at async loadESM (node:internal/process/esm_loader:*:*)
    at async handleMainPromise (node:internal/modules/run_main:*:*)
